src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
     @Override
     public void add(T value) {
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor directly instead of setting them in the next lines.

Reply:
I need them. Without them code will not working